exercise_09/getDistanceMessageFromSumTo100.js

```diff
@@ -19,3 +19,21 @@
  * random (generated in function): 40
  * output: "Sum with value 70 isd left in 30 from number 100"
  */
+
+/**
+ * 
+ * @param {number} value 
+ * @returns {string}
+ */
+function getDistanceMessageFromSumTo100(value) {
+    const randomNum = Math.floor(Math.random() * 101);
+    const sum = value + randomNum;
+    if (sum > THRESHOLD) {}
```

Comment:
Same here... 🤔

- [ ] Remove `else` statement, according to [Object Calisthenics rules](https://medium.com/@davidsen/clean-code-object-calisthenics-f6f4dec07c8b).